packages/common/src/utils/decimal.ts

```diff
@@ -54,3 +54,10 @@ export const decimalIntegerFromHumanReadable = (
 ) => {
   return parseFloat(value) * 10 ** precision
 }
+
+export const getDecimalPlaces = (priceUSD: number) => {
```

Comment:
how about `getPriceDecimalPlaces`? Also could make this accept optional arg and return 2 here if arg is undefined.